design-system/BottomSheet/BottomSheetModal.tsx

```diff
@@ -40,8 +43,15 @@ export const BottomSheetModal = memo(
           containerComponent={
             Platform.OS === "ios" ? renderContainerComponent : undefined
           }
+          enableDynamicSizing={false} // By default we don't want enable dynamic sizing
```

Comment:
This was the fix to stop the bottom sheet from going to a snapPoint that didn't exist

Reply:
@thierryskoda good catch! I initially preferred a dynamic height to let the drawer expand with reactions, but I'm fine with a fixed height! :)